src/homepageExperience/components/DataListening.tsx

```diff
@@ -78,6 +97,16 @@ class DataListening extends PureComponent<Props, State> {
   }
 
   public render() {
+    if (
+      this.props.bucket === 'sample-bucket' ||
+      this.props.bucket === DEFAULT_BUCKET
+    ) {
+      return (
+        <FlexBox justifyContent={JustifyContent.Center}>
+          <h4>Please select a bucket to listen to</h4>
```

Comment:
Added this so users aren't staring at a blank border box like this: 

<img width="1189" alt="Screen Shot 2022-09-19 at 3 41 30 PM" src="https://user-images.githubusercontent.com/18511823/191132014-d9d95271-a954-4a80-9791-1d5b50f05617.png">

Reply:
instead they see this: 

<img width="1407" alt="Screen Shot 2022-09-19 at 3 42 04 PM" src="https://user-images.githubusercontent.com/18511823/191132072-efd1c877-6b66-426a-847d-f9955ebbdcca.png">

Reply:
@taramk wanted to run this by you.